src/@types/requests.ts

```diff
@@ -47,7 +47,13 @@ export interface IJoinRoomOpts {
 
 export interface IRedactOpts {
     reason?: string;
-    // also delete related events, if the server supports it (MSC3912)
+    /**
+     * Also try to delete related events of the relation types in this array.
```

Comment:
I think we should be more explicit about what this actually means:

```suggestion
     * Whether events related to the redacted event should be redacted.
     *
     * If specified, then any events which relate to the event being redacted with
     * any of the relationship types listed will also be redacted.
     *
```

Reply:
Thanks for your suggestion :+1: Updated accordingly